include/SDL3/SDL_process.h

```diff
@@ -43,8 +43,8 @@ typedef enum {
     SDL_PROCESS_STDOUT = 1 << 1,
     /** Create a pipe from the process' stderr. Without this option, the process will output to the parent's stderr. */
     SDL_PROCESS_STDERR = 1 << 2,
-    /** Allow SDL to report errors on the child process' stderr if launching the process failed after a fork(). */
-    SDL_PROCESS_ERRORS_TO_STDERR = 1 << 3,
+    /** Forward sterr to stdout. Requires SDL_PROCESS_STDOUT and SDL_PROCESS_STDERR. */
+    SDL_PROCESS_STDERR_TO_STDOUT = 1 << 3,
 } SDL_ProcessFlags;
```

Comment:
Would it be worth making this flag `1 << 3 | SDL_PROCESS_STDOUT | SDL_PROCESS_STDERR` to avoid extra code checks and long flag lists?

Reply:
Perhaps, I applied your suggestion. I'm not sure.
Further review, and extra tests, should refine the API.